README.md

```diff
@@ -1 +1,66 @@
-# PersonalProject
\ No newline at end of file
+# The High Score Tracker
+
+## Description
+The High Score tracker will allow a program to digitally keep track of high scores without the need for memorization or pen & paper!
+
+## Overview
+The High Score Tracker keeps a record of the high scores for a game. After a game ends, the player will enter their name and score using their keyboard. The player’s score is added to a high score list in the correct location. Lastly, the updated high score list is output to the console. 
+
+## High-Level Design
+### Flowchart
+![A Flowchart](Flowchart.png)
+
+### Methods:
+I have broken down the program into 4 methods:
+- static List<int> LoadScoreValues(string filename);
+- static int GetScoreValue(string score);
+- static int FindInsertionPoint(List<int> values, int newScore);
+- static void AddScore(String name, int score, int insertAt, List<String> scores);
+
+### List LoadScoreValues(string filename)
+The scores.txt file is stored in two columns. The first column is the players name and the second column is the players score. This method loads the file and extracts the second column into a ```List<int>.```
+
+![A Flowchart] (flowcharts/loadscorevalues.png)
+
+1. Check to see if the specified filename exists.
+2. If it does not, throw an exception stating the file cannot be loaded
+3. Load the file as a List and store it in a variable named, "lines"
+4. Create a List called values.
+5. Loop through each line
+   - For each line, use GetScoreValue to extract the second column as an int
+   - Add the int to values
+6. Return values
+
+### int GetScoreValue(string score)
+This is a helper method which, given a score string of the format "name score", extracts the score value from the second column, parses it into an int value, and finally returns it.
+
+Flowchart
```

Comment:
The image is in your flowchart directory. Make sure to add them to this document.